javascript/eslint-config-react/index.js

```diff
@@ -0,0 +1,84 @@
+module.exports = {
+  env: {
+    browser: true,
+  },
+  extends: ["plugin:prettier/recommended", "react-app"],
+  plugins: ["prettier", "react-hooks", "import"],
+  rules: {
+    "arrow-body-style": ["error", "as-needed"],
+    "arrow-parens": "off",
+    "comma-dangle": [
+      "error",
+      {
+        arrays: "always-multiline",
+        exports: "always-multiline",
+        functions: "never",
+        imports: "always-multiline",
+        objects: "always-multiline",
+      },
+    ],
+    "function-paren-newline": "off",
+    "import/default": "error",
```

Comment:
pridame aj https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-cycle.md ? cas od casu taketo cosi robi neplechu (specialne pri produkcnych buildoch sa to zvykne spravat inak)